tests/ci_requirements.txt

```diff
@@ -9,8 +9,4 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 # This file enumerates various third-parties required for continuous integration.
-
-# This is required by coveralls.io, which is used to upload our test coverage
-# results.
-coverage==4.4.1
-coveralls==1.1
+coverage==4.5.4
```

Comment:
4.5.x series has support for the `[paths]` section, which allows to match coverage for files found at different locations.